scripts/Tools/compare_test_results

```diff
@@ -66,6 +66,12 @@ formatter_class=argparse.ArgumentDefaultsHelpFormatter
 
     CIME.utils.setup_standard_logging_options(parser)
 
+    parser.add_argument("-n", "--namelists-only", action="store_true",
+                        help="Only analyze namelists.")
+
+    parser.add_argument("--hist-only", action="store_true",
```

Comment:
Ah right, of course. I'd still vote for a one-character equivalent like '-i' (the second letter of 'hist'), but don't feel strongly about that.